src/main/java/com/mraof/minestuck/entity/MSEntityTypes.java

```diff
@@ -101,6 +104,9 @@ public static void registerEntities(final RegistryEvent.Register<EntityType<?>>
 		register(registry, EntityType.Builder.<SbahjPosterEntity>create(SbahjPosterEntity::new, EntityClassification.MISC).size(0.5F, 0.5F).setShouldReceiveVelocityUpdates(false).setTrackingRange(10).setUpdateInterval(Integer.MAX_VALUE), "sbahj_poster");
 		register(registry, EntityType.Builder.<ShopPosterEntity>create(ShopPosterEntity::new, EntityClassification.MISC).size(0.5F, 0.5F).setShouldReceiveVelocityUpdates(false).setTrackingRange(10).setUpdateInterval(Integer.MAX_VALUE), "shop_poster");
 		register(registry, EntityType.Builder.<HologramEntity>create(HologramEntity::new, EntityClassification.MISC).size(0.25F, 0.25F).setShouldReceiveVelocityUpdates(false).setTrackingRange(10).setUpdateInterval(Integer.MAX_VALUE), "hologram");
+		
+		register(registry, EntityType.Builder.<ConsumableProjectileEntity>create(ConsumableProjectileEntity::new, EntityClassification.MISC).size(0.25F, 0.25F).setTrackingRange(4).setUpdateInterval(10), "consumable_projectile");
+		register(registry, EntityType.Builder.<ReturningProjectileEntity>create(ReturningProjectileEntity::new, EntityClassification.MISC).size(0.25F, 0.25F).setTrackingRange(6).setUpdateInterval(1), "returning_projectile");
```

Comment:
What's the background to the tracking range and update interval values?

Reply:
The change in tracking range is probably unnecessary, but the update interval values seem to affect how visible it is. A high update interval seems to cause it to visually pass through blocks or appear invisible making it hard to keep track of

Reply:
Should I make any changes to the values I made?

Reply:
Eh, it was more of noticing how the values didn't seem to be picked from any similar minestuck or vanilla entity type, so I want to make sure that they weren't just arbitrarily picked.

Reply:
Eh, it was more of noticing how the values didn't seem to be picked from any similar minestuck or vanilla entity type, so I want to make sure that they weren't just arbitrarily picked.